helm-chart/renku/values.yaml

```diff
@@ -116,14 +118,55 @@ ingress:
 
 ## Keycloak configuration
 keycloak:
+
+  ## Spawn a keycloak instance
+  enabled: true
+
+  postgresql:
+    # Disable PostgreSQL dependency
+    enabled: false
+
+  extraEnv: |
+    - name: DB_VENDOR
+      value: postgres
+    - name: DB_PORT
+      value: "5432"
+    - name:  PROXY_ADDRESS_FORWARDING
+      value: "true"
+
+
+  # The following environment variables are provided to keycloak
+  # as extraEnvFrom secrets.
+
+  # renku-keycloak-postgres
+  # - DB_ADDR: database host                            value: <RELEASE-NAME>-postgresql
+  # - DB_DATABASE: database name for keycloak           value: global.keycloak.postgresDatabase
+  # - DB_USER: database username for keycloak database  value: global.keycloak.postgresUser
+  # - DB_PASSWORD: database password for keycloak       value: global.keycloak.postgresPassword.value
+  # keycloak-password-secret
+  # - KEYCLOAK_USER: keyclaok admin username            value: global.keycloak.user
+  # - KEYCLOAK_PASSWORD: keycloak admin password        value: global.keycloak.password.value
+
+  extraEnvFrom: |
+    - secretRef:
+        name: renku-keycloak-postgres
+    - secretRef:
+        name: keycloak-password-secret
+
+  extraVolumeMounts: |
+    - name: theme
+      mountPath: /opt/jboss/keycloak/themes/renku-theme
+
+  extraVolumes: |
+    - name: theme
+      emptyDir: {}
+
+
   ## Create a demo user in keycloak? Note that the password for the demo
   ## user must be queried from kubernetes (see the rendered NOTES.txt
   ## template which is shown after a successul deployment).
   createDemoUser: false
 
-  ## Spawn a keycloak instance
-  enabled: true
```

Comment:
dont we need this anymore? what about deployments with external keycloak?

Reply:
It's still there, just moved a bit.